docs/DeveloperGuide.md

```diff
@@ -333,9 +422,139 @@ should be able to accomplish most of the tasks faster using commands than using
+       
+       ![List All Archived Clients](images/listAllArchivedClients.png)
+
+### Archiving a client
+
+1. Archiving a client while active clients are being shown
+
+   1. Prerequisites: List all active clients (see [List Active Clients](#listing-all-active-clients)). At least one active client.
+
+   2. Test case: `archive 1`<br>
+      Expected: 
+      First client is archived, and is no longer shown in the active list. Details of the archived client shown in the status message.
+      Upon listing all archived clients (see [List Archived Clients](#listing-all-archived-clients)), the archived client can be seen.
+
+   3. Test case: `archive 0`<br>
+      Expected: No client is archived. Error details shown in the status message.
+
+   4. Other incorrect archive commands to try: `archive`, `archive x`, `...` (where x is larger than the active list size)<br>
+      Expected: Similar to previous.
+
+
+### Unarchiving a client
+
+1. Unarchiving a client while archived clients are being shown
+
+   1. Prerequisites: List all archived clients (see [List Archived Clients](#listing-all-archived-clients)). At least one archived client.
+
+   2. Test case: `unarchive 1`<br>
+      Expected: 
+      First client is unarchived, and is no longer shown in the archive. Details of the unarchived client shown in the status message.
+      Upon listing all active clients (see [List Active Clients](#listing-all-active-clients)), the unarchived client can be seen.
+
+   3. Test case: `unarchive 0`<br>
+      Expected: No client is unarchived. Error details shown in the status message.
+
+   4. Other incorrect unarchive commands to try: `unarchive`, `unarchive x`, `...` (where x is larger than the archive size)<br>
+      Expected: Similar to previous.
+
+
+### Deleting a client
+
+1. Deleting a client while active clients are being shown
+
+   1. Prerequisites: List all active clients (see [List Active Clients](#listing-all-active-clients)). At least one active client.
+
+   2. Test case: `delete 1`<br>
+      Expected: 
+      First client is deleted, and is no longer shown in the active list. Details of the deleted client shown in the status message.
+      
+   3. Test case: `delete 0`<br>
+      Expected: No client is deleted. Error details shown in the status message.
+
+   4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the active list size)<br>
+      Expected: Similar to previous.
+   
+   5. Another similar test to try: Deleting a client while archived clients are being shown
+     
+
+### Displaying help
+ 
+ 1. Displaying help
+ 
+    1. Prerequisites: Internet access.
+
+    2. Test case: `help`<br>
+       Expected: Success message shown. Also shows a popup explaining how to access the help page (User Guide). The URL can be copied with "Copy URL".
+       
+       ![help message](images/helpMessage.png)
+       
+       Upon pasting the URL into the browser, the help page is displayed.
```

Comment:
small change, instead of saying help page, say UG

Reply:
Done, can check now